managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/OpReadEntry.java

```diff
@@ -141,8 +141,8 @@ void checkReadCompletion() {
                 cursor.ledger.startReadOperationOnLedger(nextReadPosition, OpReadEntry.this);
             }
 
-            // Schedule next read in a different thread
```

Comment:
This is a behaviour change
how can we verify that we are not breaking something or reducing overall performances ?

Reply:
> how can we verify that we are not breaking something or reducing overall performances ?

testing, testing, testing. we need more Fallout tests. :)